File: Cpp/201 - 400/319 Bulb Switcher.cpp
// 0ms, 100.00%; 6.2MB, 5.13%
class Solution
{
public:
    int bulbSwitch(int n)
    {
        int m = 1, count = 0;
        while (m * m <= n) ++m, ++count;
        return count;
    }
};

File: Python3/201 - 400/319 Bulb Switcher.py
# 24ms, 96.07%; 13.9MB, 36.25%
class Solution:
    def bulbSwitch(self, n: int) -> int:
        m, count = 1, 0
        while m * m <= n:
            m += 1
            count += 1
        return count
